### PR TITLE
pa-exercise

### PR DESCRIPTION
**Pull request by OmarHafsa**

### CODE REVIEW

File: main.C [hunk 1]
```diff
@@ -18,5 +18,6 @@ int main()
         return 1;
     }
     remplirTableauAvecEntiersAleatoires(t, size, 1000000);
+    TriSelection(t, size);
```
Comment (Meriem1551) [inline main.C]: i will try to run it

File: tri_selection.c [hunk 2]
```diff
@@ -0,0 +1,24 @@
+#include "main.h"
+#include <stdlib.h>
+#include <stdio.h>
+void TriSelection(int *tab, int size)
+{
+    for (int i = 0; i < size-1; i++)
+    {
+      int index =i;
```
Comment (Meriem1551) [inline tri_selection.c]: name this i-min

File: tri_selection.c [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+	  for(int j = i+1; j < size; j++)	
+        {
+       	 if(tab[index] > tab[j])
+		    {
+       	 	index = j;
+		    }
+	    }
+	  if(index != i)
```
Comment (Meriem1551) [inline tri_selection.c]: remove this test, we dont need it

File: tri_selection.c [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	   {
+	   	int m;
```
Comment (Meriem1551) [inline tri_selection.c]: declaration at the top of the function

File: main.C (hunk 1 above)
Comment (OmarHafsa) [reply main.C]: can you try now please ?

**Comment by Meriem1551** [comment]
Good joooooob

**Pull request merged**